mojo/services/keyboard/interfaces/BUILD.gn

```diff
@@ -9,4 +9,10 @@ mojom("interfaces") {
   sources = [
     "keyboard.mojom",
   ]
+
+  import_dirs = [ get_path_info("../../", "abspath") ]
+
+  deps = [
+    "../../native_viewport/interfaces",
```

Comment:
The `mojo` directory is set up to be loaded at paths (e.g., either at the root or in `third_party`).  That's pretty rare for gn builds, which is why most other gn files use root-relative paths.  I'm not sure we use the ability to map the directory into `third_party` anywhere anymore.